cves/CVE-2012-0022.yml

```diff
@@ -120,10 +132,14 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
-  contest:
+  answer: |
+    Julian Wälde and Alexander Klink reported a way to degrade performance of
```

Comment:
The Tomcat security bulletin (http://tomcat.apache.org/security-5.html#Apache_Tomcat_5.x_vulnerabilities) states:
> This was identified by the Tomcat security team on 21 October 2011 and made public on 17 January 2012.

I don't see their names (or the date you provided) in any of the sources I've found.

Reply:
It was a very deep dive that took a long time to find the original source. This was a very widespread issue with handling hash maps. Tomcat was only one of the systems effected.